content/projects/linux/beginner/_index.md

```diff
@@ -13,8 +13,10 @@ submission_type: repo
 tags:
 - bash
 title: Beginner Linux challenges
+ready: true
 ---
 
+
 ### Submission guidelines
```

Comment:
heading hierarchy is off. Please make sure that the content is up to our stanards before marking it as ready